en/fieldwork/search.rst

```diff
@@ -5,5 +5,13 @@ Tap on the **Search button** on the top-right corner to search in the layers.
 1. Tap on the **name** in the result to pan the map on the feature (the feature will be highlighted in yellow).
 2. Tap on the **attributes button** to open its attributes.
 
-.. image:: ../images/user-guide_search.png
+.. image:: ../images/search.gif
 
+GoTo Coordinate
+---------------
+It is possible to go to Coordinates with the search tool. 
```

Comment:
```suggestion
It is possible to go to coordinates with the search tool. 
```

Reply:
@lucienicolier these are not yet resolved (or probably were reverted at some point)